configs/nvme-storage-configurator/daemonset.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: nvme-storage-configurator
+  labels:
+    app: nvme-storage-configurator
+  namespace: kube-system
```

Comment:
i don't think we should use system namespace. Is there a reason to use it?

Reply:
It was a mistake, it should be `nvme-storage-config`. I have fixed it.

Reply:
I have tested the reboot behaviour. After I rebooted the node, I found the NVMe Disk intact.